Library/Formula/med.rb

```diff
@@ -0,0 +1,20 @@
+class Med < Formula
+  desc "Model for Exchange of Data. Data exchange system for numerical models"
+  homepage "http://www.salome-platform.org/user-section/about/med"
+  url "http://files.salome-platform.org/Salome/other/med-3.0.8.tar.gz"
+  sha256 "0f97e9b5ae514aa53f4c0507f1002a3f520c3b959e756c38cfd9cae4a31a0689"
+
+  depends_on "cmake" => :build
+  depends_on "hdf5" => "with-mpi"
+  depends_on :mpi
+  depends_on :python
+
+  def install
+    system "cmake", ".", *std_cmake_args, "-DMEDFILE_BUILD_TESTS:BOOL=OFF"
+    system "make", "install"
+  end
+
+  test do
+    system "false"
```

Comment:
Hi Dominyk,

MED comes with a set of tests which are compiled by setting a CMAKE flag and then running make test in the build directory, this is a bit beyond my skills with Ruby so some advice would be appreciated.

David


Professor D M Ingram, PhD
Institute for Energy Systems
School of Engineering
The University of Edinburgh
Colin MacLaurin Road
Edinburgh EH9 3DW

Tel: +44 (0)131 651 9022
School fax: +44 (0)131 650 6554

Latest Papers:
L. Cradden, C. Kalgoeri, I. Martinez Barrios, G. Galanis, D. M. Ingram, and G. Kallos. Multi-criteria site selection for offshore renewable energy platforms. Renewable Energy, 87:791–806, 2016.
http://dx.doi.org/10.1016/j.renene.2015.10.035

Watanabe Y, Ingram DM (2015) Transverse instabilities of ascending planar jets formed by wave impacts on vertical walls. Proc. R. Soc. A 471:20150397.  http://dx.doi.org/10.1098/rspa.2015.0397

The University of Edinburgh is a charitable body, registered in Scotland, with registration number SC005336.

> On 26 Feb 2016, at 18:52, Dominyk Tiller <notifications@github.com> wrote:
> 
> In Library/Formula/med.rb <https://github.com/Homebrew/homebrew/pull/49566#discussion_r54284782>:
> 
> > +  homepage "http://www.salome-platform.org/user-section/about/med"
> > +  url "http://files.salome-platform.org/Salome/other/med-3.0.8.tar.gz"
> > +  sha256 "0f97e9b5ae514aa53f4c0507f1002a3f520c3b959e756c38cfd9cae4a31a0689"
> > +
> > +  depends_on "cmake" => :build
> > +  depends_on "hdf5" => "with-mpi"
> > +  depends_on :mpi
> > +  depends_on :python
> > +
> > +  def install
> > +    system "cmake", ".", *std_cmake_args, "-DMEDFILE_BUILD_TESTS:BOOL=OFF"
> > +    system "make", "install"
> > +  end
> > +
> > +  test do
> > +    system "false"
> Can this test be modified to do something more substantial than e.g. --version or --help? See cmake.rb for an example of an application formula with a good test and tinyxml2.rb for an example of a library formula with a good test. Thanks!
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Homebrew/homebrew/pull/49566/files#r54284782>.
> 


The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.